src/forms/TextInput.jsx

```diff
@@ -123,42 +81,32 @@ export const TextInput = (props: Props): React$Element<*> => {
 	}
 
 	return (
-		<div className="inputContainer">
+		<div>
 			{label && (
-				<label
-					className={classNames.label}
-					htmlFor={id}
-					data-requiredtext={required && requiredText}
-				>
+				<label htmlFor={id || name}>
 					{label}
+					{required ? <span> {requiredText}</span> : ''}
 				</label>
 			)}
-			{helperText && <div className={classNames.helperText}>{helperText}</div>}
-			<div style={{ position: 'relative' }}>
-				<input
-					type={isSearch ? 'search' : 'text'}
+
+			{helperText && <p data-swarm-select-helper-text="1">{helperText}</p>}
+
+			<div data-swarm-select-wrapper="1">
```

Comment:
& here

Reply:
Are these to be replaced with `<FieldLabel />` and `<FieldHelper />`?

Reply:
Yep, the purpose is to have something generic for styling labels and helper text.